libdnf/sack/query.cpp

```diff
@@ -1321,7 +1322,9 @@ Query::Impl::filterAdvisory(const Filter & f, Map *m, int keyname)
                     eq = false;
             }
             if (eq) {
-                advisory.getPackages(pkgs, false);
+                auto advisoryPkgs = advisory.getPackages(false);
```

Comment:
This is not only usage of that in libdnf. Cannot more help, because cannot test it.